src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -63,4 +68,17 @@ class ReferralController(private val referralService: ReferralService) {
   fun getDraftReferralsCreatedByUserID(@RequestParam userID: String): List<DraftReferral> {
     return referralService.getDraftReferralsCreatedByUserID(userID)
   }
+
+  @GetMapping("/service-category/{id}")
+  fun getServiceCategoryByID(@PathVariable id: String): ResponseEntity<Any> {
+    val uuid = try {
+      UUID.fromString(id)
+    } catch (e: IllegalArgumentException) {
+      return ResponseEntity.badRequest().body("malformed id")
```

Comment:
Is it worth adding some more detail to the error message here?

Reply:
yes - but this is handled as part of another ticket.

Reply:
👍